### DISCUSSION TITLE
Need help with prism launcher on my steam deck!

### DISCUSSION THREAD

**Discussion by TRTGGE** [PrismLauncher/PrismLauncher#2424] [Q&A]
So I don't know if someone can point me in the right direction but I seem to be having a problem with adding my account since I press add microsoft and then I just get a box come up saying working with a blue bar filled to about 15%. But It seems like nothing happens. On another note is that I was able to vertify my account and connect it up but was getting another issue, Which was saying "Failed to get minecraft access token: operation canceled". Honestly  just had enough since I have been trying to get this to work now for 2 days with nothing but problems so am looking for some help or pointing me in the right direction. If you need any extra information let me know thanks! 

Any help is appreciated! :) 

Edit: 19/05/2024 16:17

Another quick possible bug is that sometimes when it does go through to the browser and puts the code in and you sign into your microsoft account it will just say it is connected rather than asking if you want to let this app access your info? I find when it goes through with asking for permission there is a higher chance of not getting the error above which is " Failed to get minecraft access token: operation canceled".
Again it does not happen all the time just seems a higher chance you don't get that error.

**Comment by Trial97** [comment]
For all the stuff related to login see https://github.com/PrismLauncher/PrismLauncher/issues/2302

**Reply by TRTGGE** [reply]
Thanks I will look through it and get back to you! :)

**Reply by TRTGGE** [reply]
Hi would you rather me respond here with what I have tried or in the thread you have linked? Because I have tried alot of the solutions on my steam deck and it seems to not have worked. Some of the solutions I have searched for like the firewall etc and it seems steam deck does not have one? Or it is not possible to access it like windows fire wall.

**Reply by TRTGGE** [reply]
Also another thing is that every now and again it will allow me to add my account but after a certain amount of time or if I shutdown the prism launcher it will show ,u account status as offline.

**Reply by roke-julian-lockhart** [reply]
> https://github.com/orgs/PrismLauncher/discussions/2424#discussioncomment-9489127

@TRTGGE, best to ask such (more broad, off-topic) questions at https://www.reddit.com/r/SteamDeck/submit (or https://steamcommunity.com/app/1675200/discussions/). However, it does have a firewall, irrespective of whether a GUI for it comes preinstalled. 3rd-party applications to control the firewall do exist. Since the Steam Deck uses KDE Plasma 6 as its desktop environment, I know that a KCM (a module for `systemsettings` exists) as does a separate GTK application.

**Reply by Ari-43** [reply]
> Hi would you rather me respond here with what I have tried or in the thread you have linked? Because I have tried alot of the solutions on my steam deck and it seems to not have worked. Some of the solutions I have searched for like the firewall etc and it seems steam deck does not have one? Or it is not possible to access it like windows fire wall. 

The Steam Deck does have a firewall, but that's unlikely to be a firewall issue since I assume other Internet services work normally on the device.

**Reply by roke-julian-lockhart** [reply]
> https://github.com/orgs/PrismLauncher/discussions/2424#discussioncomment-9920770

@Ari-RERA-43, that's quite an assumption. As an example, any new installation of [openSUSE Tumbleweed](https://download.opensuse.org/tumbleweed/iso/openSUSE-Tumbleweed-NET-x86_64-Snapshot20240629-Media.iso?mirrorlist) blocks Minecraft due to firewall rules, yet everything else appears to work (in my experience). It's not unreasonable to expect that the firewall is the most probable culprit.
